TemplateApplication/Contacts/Contacts.swift

```diff
@@ -10,8 +10,12 @@ import Foundation
 import SpeziContact
 import SwiftUI
 
+/// Display contacts for your app.
+///
+/// Modify the contact details for your app in the Contacts view.
```

Comment:
Very nice! It might even make sense to link to the relevant module documentations here. Also applies to other files.